hugo.yaml

```diff
@@ -6,7 +6,6 @@ pygmentsUseClasses: true
 disableKinds: ["taxonomy"]
 canonifyURLs: true
 theme: ["basetheme", "jaeger-docs"]
-googleAnalytics: "UA-148208642-1"
```

Comment:
shouldn't we upgrade it instead of dropping? iiuc the Hugo configuration is still the same and accept different versions of GA ID.

Reply:
I've configured it via Netlify. For details, see #886.

Reply:
> <img width="733" alt="image" src="https://github.com/user-attachments/assets/d32156bd-22e1-46b0-a780-49d602860cdb" />